robot/2023/src/main/java/frc/robot/Path.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot;
+
+public class Path extends Robot
+{
+    //these may need to be floats depending on the encoder, I'll just wait and see though
+    double target;
+    double wait;
+    Robot bot;
+    public Path(double target, double wait, Robot bot)
+    {
+        this.target = target;
+        this.wait = wait;
+        this.bot = bot;
+    }
+    public void tick() {
```

Comment:
Need method comment.  Something like:
/*
tick() is run every 20ms and:
1. Measures what distance we've actually travelled.
2. Sets the motor to some reasonable rate (50%, 10%, etc) depending on how far away we are from target.
3. If wait > 0, we may just decrement wait.
*/

Reply:
Basically just copied that over